docs/Security-Compliance/Security-Controls/ac-2-3/index.md

```diff
@@ -0,0 +1,14 @@
+# Access Control
+## AC-02(03) - DISABLE INACTIVE ACCOUNTS  
+
+The information system automatically disables inactive accounts after [a maximum of 180 days of inactivity].  
+
+For CSP Only: AC-2 (3) [90 days for user accounts]. 
+AC-2 (3) Additional FedRAMP Requirements and Guidance:  
+Requirement: The service provider defines the time period for non-user accounts (e.g., accounts associated with devices).  The time periods are approved and accepted by the Joint Authorization Board (JAB)/AO. Where user management is a function of the service, reports of activity of consumer users shall be made available.
+
+### TDP Implementation  
+
+The System Admin reviews the list of TDP application users on a monthly basis and/or when the status of a user is changed to inactive or terminated. Inactive accounts are removed from the system manually if their status has changed.
+
```

Comment:
This looks great and makes sense to me - I'm wondering if it might be helpful to include that Django Admin provides user activity logs that we can use to validate whether a particular user has been active or not. Would that be accurate?

Reply:
Ok, I have verified it. You can see the date the user last logged in in their user profile on Django Admin. I will document that.

Reply:
@hursey013 this is done